### PR TITLE
Add bluetooth readout

### PR DESCRIPTION
**Pull request by triveria**
Hi,
this MR implements a simple (but not so elegant) method to read out the controller via bluetooth.
I haven't found a more elegant method yet.
It works on Linux. I haven't tested it on Windows.

### CODE REVIEW

File: examples/read_all_input_channels.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    print(f"epoch: {time.time():.0f}")
+    time.sleep(0.5)
+
+
+stdscr = curses.initscr()
+curses.noecho()
+curses.cbreak()
+
+
+while True:
```
Comment (flok) [inline examples/read_all_input_channels.py]: could you add some option to gracefully close the curses display and stop the example? maybe by pressing q ?

File: examples/read_all_input_channels.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    stdscr.addstr(15,  0, f"touchFinger1: {dualsense.state.touchFinger1} \t touchFinger2: {dualsense.state.touchFinger2}")
+    stdscr.addstr(16,  0, f"micBtn: {dualsense.state.micBtn}")
+    stdscr.addstr(17,  0, f"RX: {dualsense.state.RX} \t RY: {dualsense.state.RY} \t LX: {dualsense.state.LX} \t LY: {dualsense.state.LY}")
+    stdscr.addstr(18,  0, f"trackPadTouch0: ID: {dualsense.state.trackPadTouch0.ID} \t isActive: {dualsense.state.trackPadTouch0.isActive} \t X: {dualsense.state.trackPadTouch0.X} \t Y: {dualsense.state.trackPadTouch0.Y}")
+    stdscr.addstr(19,  0, f"trackPadTouch1: ID: {dualsense.state.trackPadTouch1.ID} \t isActive: {dualsense.state.trackPadTouch1.isActive} \t X: {dualsense.state.trackPadTouch1.X} \t Y: {dualsense.state.trackPadTouch1.Y}")
+    stdscr.addstr(20, 0, f"gyro: roll: {dualsense.state.gyro.Roll} \t pitch: {dualsense.state.gyro.Pitch} \t yaw: {dualsense.state.gyro.Yaw}")
+    stdscr.addstr(21, 0, f"acc: X: {dualsense.state.accelerometer.X} \t Y: {dualsense.state.accelerometer.Y} \t Z: {dualsense.state.accelerometer.Z}")
+
+    stdscr.refresh()
+    # time.sleep(0.1)
```
Comment (flok) [inline examples/read_all_input_channels.py]: not necessary comment

**Review by flok — CHANGES_REQUESTED** [review]

**Comment by flok** [comment]
Thanks for the PR! Added some little comments before merging

**Comment by triveria** [comment]
Thanks for your suggestions. I've incorporated them now

**Review by flok — APPROVED** [review]

**Pull request merged**